applications/luci-app-haproxy-tcp/luasrc/view/haproxy/haproxy_status.htm

```diff
@@ -0,0 +1,22 @@
+<script type="text/javascript">//<![CDATA[
+XHR.poll(3, '<%=url([[admin]], [[services]], [[haproxy]], [[status]])%>', null,
+	function(x, data) {
+		var tb = document.getElementById('haproxy_status');
+		if (data && tb) {
+			if (data.running) {
+				var links = '<em><b><font color=green>HAProxy <%:RUNNING%></font></b></em>';
```

Comment:
This variable `links` is not referred any elsewhere. So why make it? Why don't you make a same variable for another status?